warehouse/models/mart/payments/fct_elavon__deposit_billing_transactions.sql

```diff
@@ -0,0 +1,78 @@
+{{ config(materialized='table') }}
+
+WITH
+
+int_elavon__billing_transactions_deduped AS (
+    SELECT * FROM {{ ref('int_elavon__billing_transactions_deduped') }}
+),
+
+int_elavon__deposit_transactions_deduped AS (
+    SELECT * FROM {{ ref('int_elavon__deposit_transactions_deduped') }}
+),
+
+union_deposits_and_billing AS (
+
+    SELECT
+        *
+    FROM int_elavon__billing_transactions_deduped
+    UNION ALL
+    SELECT
+        *
+    FROM int_elavon__deposit_transactions_deduped
+
+),
+
+fct_elavon__deposit_billing_transactions AS (
+
+    SELECT
+
+        payment_reference,
+        payment_date,
+        account_number,
+        routing_number,
+        fund_amt,
+        batch_reference,
+        batch_type,
+        customer_batch_reference,
+        customer_name,
+        merchant_number,
+        external_mid,
+        store_number,
+        chain,
+        batch_amt,
+        amount,
+        surchg_amount,
+        convnce_amt,
+        card_type,
+        charge_type,
+        charge_type_description,
+        card_plan,
+        card_no,
+        chk_num,
+        transaction_date,
+        settlement_date,
+        authorization_code,
+        chargeback_control_no,
+        roc_text,
+        trn_aci,
+        card_scheme_ref,
+        trn_ref_num,
+        settlement_method,
+        currency_code,
+        cb_acq_ref_id,
+        chgbk_rsn_code,
+        chgbk_rsn_desc,
+        mer_ref,
+        purch_id,
+        cust_cod,
+        trn_arn,
+        term_id,
+        ent_num,
+        dt,
+        execution_ts
+
+    FROM union_deposits_and_billing
+
+)
+
+SELECT * FROM fct_elavon__deposit_billing_transactions
```

Comment:
I think that *in general* it is preferable that mart tables maybe don't have a specific vendor name, and instead just reflect the concept. I think it's probably fine in this case, but could be something to consider in future (like, should we have `fare_processor_transactions` and `fare_calculator_transactions`? Something more generic and vendor-agnostic.) Maybe just longer-term food for thought